### ISSUE TITLE
Could you share your pretrained model? Thank you!

### ISSUE THREAD

**Issue by Eleanor456** [JoeDelK/A-learning-approach-for-garbage-and-debris-identification#1]
Thank you for your working about the garbage and debris identification. I am very interested about it too, could you upload your pretrained model for test. Thank you very much!

**Comment by JoeDelK** [comment]
@Eleanor456 Yes, sure! do you have any way of contact you? email, linkedin, etc...

**Comment by Eleanor456** [comment]
My email address is：peige1994@foxmail.com


Thank you.




------------------ Original ------------------
